CMakeLists.txt

```diff
@@ -475,6 +475,17 @@ if (NOT DEFINED MFEM_TIMER_TYPE)
   endif()
 endif()
 
+# ALE3D begin edit
```

Comment:
I don't think we need the Begin ALE3D Edits and end ALE3D Edits tags.  Perhaps a comment on

Reply:
Sure, I can remove those.